modkit.wdl

```diff
@@ -0,0 +1,107 @@
+version 1.0
+
+# Copyright (c) 2025 Leiden University Medical Center
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+task Pileup {
+    input {
+        File bam
+        File bamIndex
+        String outputBed = "output.bed"
+        File referenceFasta
+        File referenceFastaFai
+
+        Int? intervalSize
+        File? includeBed
+
+        Boolean cpg = false
+        Boolean combineMods = false
+        Boolean combineStrands = false
+        Boolean bedgraph = false
+        String? ignore
+        String logFilePath = "modkit.log"
+
+        Int threads = 8
+        String memory = "48GiB"
+        Int timeMinutes = 4320  # 3 Days
```

Comment:
Does it really need that much time?

Reply:
```
$ seff -j 20167230
Job ID: 20167230
State: COMPLETED (exit code 0)
Nodes: 1
Cores per node: 8
CPU Utilized: 02:23:31
CPU Efficiency: 65.59% of 03:38:48 core-walltime
Job Wall-clock time: 00:27:21
Memory Utilized: 1.19 GB
Memory Efficiency: 2.49% of 48.00 GB
```

Erm... Yes these estimates are very pessimistic :P. I will put in some more realistic ones.